### ISSUE TITLE
Feature request: Export file on submission

### ISSUE THREAD

**Issue by johnbleck3** [formtools/module-export_manager#14]
File exports on submission would be awesome. as xml, csv, plain text. To local folder - or - to an ftp might be asking too much, but would also be awesome.

I can't tell you how excited I was to see this app get updated! Awesome job so far.

**Comment by benkeen** [comment]
Thanks @johnbleck3 :D It's coming along... beta soon, I hope.

For the feature request, you mean the option to export the `individual` submission? Like maybe an option on the edit/view submission page that lets you download the full info in whatever format you want? [(S)FTP would be neat]

**Comment by johnbleck3** [comment]
Yes, the individual submission info to file with sftp options - even just to local to pull down would be crazy useful.

**Comment by benkeen** [comment]
Yeah, great idea. Noted!
